Lib/test/test_syntax.py

```diff
@@ -577,11 +577,11 @@
 
 >>> lambda a,*b=3,c: None
 Traceback (most recent call last):
-SyntaxError: var-positional parameter cannot have default value
+SyntaxError: var-positional parameter's default value cannot be changed because it has the immutable default value ()
```

Comment:
```suggestion
SyntaxError: Cannot change the default value of a var-positional parameter because it is already defined.
```

@AA-Turner Maybe this would make more sense to people?
For me, “immutable default value” feels too detailed, and the current message isn’t very user-friendly.